src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -244,6 +244,8 @@ trait IterableOps[+A, +CC[X], +C] extends Any {
   /** A view representing the elements of this collection. */
   def view: View[A] = View.fromIterator(coll.iterator())
 
+  def transform[B](vt: Iterable[A] => View[B]): CC[B] =  fromIterable(vt(coll))
```

Comment:
Actually, whether it ends up as a `View`, depends only on whether it started out as one, which is nice.